plan/match_property.go

```diff
@@ -258,6 +258,21 @@ func (p *Projection) matchProperty(_ *requiredProperty, childPlanInfo ...*physic
 }
 
 // matchProperty implements PhysicalPlan matchProperty interface.
+func (p *Analyze) matchProperty(_ *requiredProperty, childPlanInfo ...*physicalPlanInfo) *physicalPlanInfo {
```

Comment:
Yes, It may have more than one children.An `Analyze` statement could analyze multi-tables at once, and for each table, it may need more than one select.